src/libs/actions/Policy.ts

```diff
@@ -3565,6 +3565,37 @@ function enablePolicyTags(policyID: string, enabled: boolean) {
 }
 
 function enablePolicyTaxes(policyID: string, enabled: boolean) {
+    const taxRatesData: OnyxData = {
+        optimisticData: [
+            {
+                onyxMethod: Onyx.METHOD.MERGE,
+                key: `${ONYXKEYS.COLLECTION.POLICY}${policyID}`,
+                value: {
+                    taxRates: DEFAULT_TAXES_OPTIMISTIC_DATA,
+                },
+            },
+        ],
+        successData: [
+            {
+                onyxMethod: Onyx.METHOD.MERGE,
+                key: `${ONYXKEYS.COLLECTION.POLICY}${policyID}`,
+                value: {
+                    taxRates: DEFAULT_TAXES_SUCCESS_DATA,
+                },
+            },
+        ],
+        failureData: [
+            {
+                onyxMethod: Onyx.METHOD.MERGE,
+                key: `${ONYXKEYS.COLLECTION.POLICY}${policyID}`,
+                value: {
+                    taxRates: undefined,
+                },
+            },
+        ],
+    };
+    const policy = ReportUtils.getPolicy(policyID);
+    const shouldAddDefaultTaxRatesData = isEmptyObject(policy.taxRates) && enabled;
```

Comment:
`ReportUtils.getPolicy` is deprecated and it also seems like `policy.taxRates` can be [undefined](https://github.com/Expensify/App/blob/e2bff569d49b8a740721f007a8ef441b3d82767b/src/types/onyx/Policy.ts#L332)
```suggestion
    const policy = allPolicies?.[`${ONYXKEYS.COLLECTION.POLICY}${policyID}`];
    const shouldAddDefaultTaxRatesData = (!policy?.taxRates || isEmptyObject(policy.taxRates)) && enabled;
```